### DISCUSSION TITLE
Minimal HA Cluster instance size

### DISCUSSION THREAD

**Discussion by KlausVii** [gravitational/teleport#17723] [Q&A] [answered]
Hiya,

I'm following https://github.com/gravitational/teleport/tree/master/examples/aws/terraform/ha-autoscale-cluster to create an HA cluster and it all seems to be working. However the default instances are way too large for our use case of a couple of k8s clusters and less than 10 users. 

What's the minimal instance sizes for the various components?

**Comment by webvictim** [comment, marked as answer]
You should be able to use a couple of `t3.medium` instances for a small
cluster with just a few users. `micro` is probably just a little too small. `small` may be OK too.

**Reply by KlausVii** [reply]
Thanks; `t3.medium` for proxy and `t3.small` for auth seemed to work for us.
